platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/ConfigurationCacheState.kt

```diff
@@ -182,13 +182,10 @@ class ConfigurationCacheState(
     private val eventEmitter: BuildOperationProgressEventEmitter,
     private val host: ConfigurationCacheHost
 ) {
-    /**
-     * Writes the state for the whole build starting from the given root [build] and returns the set
```

Comment:
💭  The "returns" part of the javadoc was inaccurate, and the rest did not add more detail on top of the method name.